dashboard/app/models/section.rb

```diff
@@ -15,17 +14,18 @@
 #  deleted_at   :datetime
 #  stage_extras :boolean          default(FALSE), not null
 #  section_type :string(255)
+#  user_id      :integer
 #
 # Indexes
 #
-#  index_sections_on_code     (code) UNIQUE
-#  index_sections_on_user_id  (user_id)
```

Comment:
Am I right in thinking that we still get the index on `user_id` through `index_coteachers_on_user_id`? Note that this is important, as we need to need to be able to find all sections for a given teacher, e.g., for the teacher dashboard.